Containers/borgbackup/backupscript.sh

```diff
@@ -328,21 +340,70 @@ if [ "$BORG_MODE" = restore ]; then
         NEXTCLOUD_DATADIR='""'
     fi
 
-    # Restore the configuration file
-    if ! rsync --archive --human-readable -vv \
-    /tmp/borg/nextcloud_aio_volumes/nextcloud_aio_mastercontainer/data/configuration.json \
-    /nextcloud_aio_volumes/nextcloud_aio_mastercontainer/data/configuration.json; then
```

Comment:
I would personally like to keep rsync at least for local archives since as you found out borg extract does not allow for deleting existing files...

Reply:
Sure, I'll bring back the borg mount and rsync for local repos.

Reply:
Done